flow/electron.js

```diff
@@ -1,4 +1,5 @@
 import {IncomingMessage} from "electron"
+import {Function} from "estree"
```

Comment:
I believe you don't need to import anything and estree is some JS AST representation format